src/main/kotlin/javabot/operations/KarmaOperation.kt

```diff
@@ -112,8 +112,9 @@ class KarmaOperation @Inject constructor(bot: Javabot, adminDao: AdminDao, var d
         val message = event.value
         val sender = event.user
         if (message.startsWith("karma ")) {
-            val nick = message.substring("karma ".length).toLowerCase()
-            val karma = dao.find(nick)
+            val nick = message.substring("karma ".length)
+            val normalizedNick=nick.toLowerCase()
```

Comment:
ditto.  the search already ignores case.  lowercasing it is redundant.  ( i know it was that way before but it was wrong then, too)